companion/discord_bot/utils.py

```
@@ -0,0 +1,425 @@
+    text = re.sub(r'<@&(\d+)>', lambda m: f'<@&\u200b{m.group(1)}>', text)
+
+    return text
+
+
+def get_user_avatar_url(user: Union[User, Member]) -> Optional[str]:
+    """
+    Get the user's avatar URL.
+    """
+    try:
+        return str(user.avatar.url) if user.avatar else str(user.default_avatar.url)
+    except Exception:
```

Ruff (0.13.2)

Do not catch blind exception: `Exception`

(BLE001)